provider_sources/_compress_providers.py

```diff
@@ -84,131 +84,116 @@ def update_year(provider_or_tile):
+        "ORTHOIMAGERY.ORTHOPHOTOS": "orthos"
+    }
+
+    if variant in variant_to_name:
+        name = variant_to_name[variant]
+
+    # Get layer style
+    style = layer.get("Style")
+    if isinstance(style, dict):
+        style = style.get("ows:Identifier")
+
+    elif isinstance(style, list):
+        style = style[1].get("ows:Identifier") if len(style) > 1 else None
+    else:
+        style = "normal"
+
+    # Resolution levels (pyramid)
+    TileMatrixSet = layer["TileMatrixSetLink"]["TileMatrixSet"]
+
+    # Zoom levels
+    TileMatrixSetLimits = layer["TileMatrixSetLink"]["TileMatrixSetLimits"][
+        "TileMatrixLimits"
+    ]
+    min_zoom = int(TileMatrixSetLimits[0]["TileMatrix"])
+    max_zoom = int(TileMatrixSetLimits[-1]["TileMatrix"])
+
+    # Tile format
+    output_format = layer.get("Format") # image/png...
+    if output_format == "application/x-protobuf" or output_format == "image/x-bil;bits=32":
+        continue
+
+    # Layer extent
+    bbox_lower_left = layer["ows:WGS84BoundingBox"][
+        "ows:LowerCorner"
+    ]  # given with lon/lat order
+    bbox_upper_right = layer["ows:WGS84BoundingBox"][
+        "ows:UpperCorner"
+    ]  # given with lon/lat order
+    lower_left_corner_lon, lower_left_corner_lat = bbox_lower_left.split(
+        " "
+    )
+    upper_right_corner_lon, upper_right_corner_lat = bbox_upper_right.split(
+        " "
+    )
+    bounds = [
+        [float(lower_left_corner_lat), float(lower_left_corner_lon)],
+        [float(upper_right_corner_lat), float(upper_right_corner_lon)],
+    ]
+
+    wmts_layers_list.append("GeoportailFrance." + name)
+    leaflet["GeoportailFrance"][name] = {
+        "url": """https://data.geopf.fr/wmts?SERVICE=WMTS&VERSION=1.0.0&REQUEST=GetTile&STYLE={style}&TILEMATRIXSET={TileMatrixSet}&FORMAT={format}&LAYER={variant}&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}""",
+        "html_attribution": """<a target="_blank"href="https://www.geoportail.gouv.fr/">Geoportail France</a>""",
+        "attribution": "Geoportail France",
+        "bounds": bounds,
+        "min_zoom": min_zoom,
+        "max_zoom": max_zoom,
+        "format": output_format,
+        "style": style,
+        "variant": variant,
+        "name": "GeoportailFrance." + name,
+        "TileMatrixSet": TileMatrixSet,
+        "apikey": "your_api_key_here",
+    }
+
+    # Handle broken providers
+    possibly_broken_providers = [
+        "Points_Injection_Biomethane",
+        "Ocsge_Usage_2016",
+        "Ocsge_Usage_2011",
+        "Ocsge_Couverture_2016",
+        "Ocsge_Couverture_2014",
+        "Orthoimagery_Orthophotos_Geneve",
+        "Orthoimagery_Orthophotos_Coast2000",
+        "Ocsge_Couverture_2011",
+        "Ocsge_Couverture_2002",
+        "Ocsge_Constructions_2016",
+        "Ocsge_Constructions_2014",
+        "Ocsge_Constructions_2011",
+        "Ocsge_Constructions_2002",
+        "Ocsge_Usage_2002",
+        "Ocsge_Usage_2014",
+        "Pcrs_Lamb93"
+    ]
+
```

Comment:
I would suggest this updated list of broken basemaps but maybe a global and sytematic check is needed.
```
possibly_broken_providers = [
        "Ocsge_Constructions_2016",
        "Ocsge_Constructions_2014",
        "Ocsge_Constructions_2011",
        "Ocsge_Constructions_2002",
        "Pcrs_Lamb93"
    ]